src/pages/settings/AboutPage/AboutPage.js

```diff
@@ -80,73 +80,69 @@ function AboutPage(props) {
-                                    onPress={() => item.action()}
-                                    shouldBlockSelection={Boolean(item.link)}
-                                    onSecondaryInteraction={
-                                        !_.isEmpty(item.link) ? (e) => ReportActionContextMenu.showContextMenu(CONTEXT_MENU_TYPES.LINK, e, item.link, popoverAnchor) : undefined
-                                    }
-                                    ref={(el) => (popoverAnchor = el)}
-                                    shouldShowRightIcon
-                                />
-                            ))}
-                        </View>
-                        <View style={[styles.sidebarFooter]}>
-                            <Text
-                                style={[styles.chatItemMessageHeaderTimestamp]}
-                                numberOfLines={1}
-                            >
-                                {props.translate('initialSettingsPage.readTheTermsAndPrivacy.phrase1')}{' '}
-                                <TextLink
-                                    style={[styles.textMicroSupporting, styles.link]}
-                                    href={CONST.TERMS_URL}
-                                >
-                                    {props.translate('initialSettingsPage.readTheTermsAndPrivacy.phrase2')}
-                                </TextLink>{' '}
-                                {props.translate('initialSettingsPage.readTheTermsAndPrivacy.phrase3')}{' '}
-                                <TextLink
-                                    style={[styles.textMicroSupporting, styles.link]}
-                                    href={CONST.PRIVACY_URL}
-                                >
-                                    {props.translate('initialSettingsPage.readTheTermsAndPrivacy.phrase4')}
-                                </TextLink>
-                                .
-                            </Text>
-                        </View>
-                    </ScrollView>
-                </>
-            )}
-        </ScreenWrapper>
+        <IllustratedHeaderPageLayout
+            title={props.translate('initialSettingsPage.about')}
+            onBackButtonPress={() => Navigation.goBack(ROUTES.SETTINGS)}
+            illustration={LottieAnimations.Coin}
+            shouldShowCloseButton
+            backgroundColor={themeColors.PAGE_BACKGROUND_COLORS[ROUTES.SETTINGS_ABOUT]}
+            overlayContent={overlayContent}
+        >
+            <ScrollView contentContainerStyle={[styles.flexGrow1, styles.flexColumn, styles.justifyContentBetween]}>
+                <View style={[styles.flex1]}>
+                    <View style={[styles.settingsPageBody, styles.mb6, styles.alignItemsCenter, styles.ph5]}>
+                        <Text style={[styles.baseFontStyle]}>{props.translate('initialSettingsPage.aboutPage.description')}</Text>
+                    </View>
```

Comment:
This is missing the "About Expensify" header

Reply:
@roryabraham could you plz provide a figma so I can verify "About Expnensify" header styles.

Reply:
Good spot @s77rt!

@Pujan92 I can't provide a Figma because I don't know where the mockup we need is in our Figma or how I could provide you that access. But I can tell you that the `About Expensify` text should be using `styles.textHeadline`. You'll have to make do with the screenshot to get the paddings / margins right.